packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -2,236 +2,205 @@
+  ///
+  /// If the [controller] is null, the default behavior is to
+  /// enable scrollbar dragging using the [PrimaryScrollController].
+  ///
+  /// When null, [thickness] and [radius] defaults will result in a rounded
+  /// rectangular thumb that is 8.0 dp wide with a radius of 8.0 pixels.
   const Scrollbar({
     Key? key,
-    required this.child,
-    this.controller,
-    this.isAlwaysShown = false,
-    this.thickness,
-    this.radius,
-  }) : assert(!isAlwaysShown || controller != null, 'When isAlwaysShown is true, must pass a controller that is attached to a scroll view'),
-       super(key: key);
-
-  /// The widget below this widget in the tree.
-  ///
-  /// The scrollbar will be stacked on top of this child. This child (and its
-  /// subtree) should include a source of [ScrollNotification] notifications.
+    required Widget child,
+    ScrollController? controller,
+    bool isAlwaysShown = false,
+    this.showTrackOnHover = false,
+    this.thicknessWithTrack,
+    double? thickness,
+    Radius? radius,
+  }) : super(
+    key: key,
+    child: child,
+    controller: controller,
+    isAlwaysShown: isAlwaysShown,
+    thickness: thickness ?? _kScrollbarThickness,
+    radius: radius,
+    fadeDuration: _kScrollbarFadeDuration,
+    timeToFade: _kScrollbarTimeToFade,
+    pressDuration: Duration.zero,
+  );
+
+  /// Controls if the track will show on hover and remain, including during drag.
   ///
-  /// Typically a [ListView] or [CustomScrollView].
-  final Widget child;
+  /// Defaults to false, cannot be null.
+  final bool showTrackOnHover;
 
-  /// {@macro flutter.cupertino.cupertinoScrollbar.controller}
-  final ScrollController? controller;
-
-  /// {@macro flutter.cupertino.cupertinoScrollbar.isAlwaysShown}
-  final bool isAlwaysShown;
-
-  /// The thickness of the scrollbar.
-  ///
-  /// If this is non-null, it will be used as the thickness of the scrollbar on
-  /// all platforms, whether the scrollbar is being dragged by the user or not.
-  /// By default (if this is left null), each platform will get a thickness
-  /// that matches the look and feel of the platform, and the thickness may
-  /// grow while the scrollbar is being dragged if the platform look and feel
-  /// calls for such behavior.
-  final double? thickness;
-
-  /// The radius of the corners of the scrollbar.
+  /// The thickness of the scrollbar when a hover state is active and
+  /// [showTrackOnHover] is true.
   ///
-  /// If this is non-null, it will be used as the fixed radius of the scrollbar
-  /// on all platforms, whether the scrollbar is being dragged by the user or
-  /// not. By default (if this is left null), each platform will get a radius
-  /// that matches the look and feel of the platform, and the radius may
-  /// change while the scrollbar is being dragged if the platform look and feel
-  /// calls for such behavior.
-  final Radius? radius;
+  /// Defaults to 12.0 dp when null.
+  final double? thicknessWithTrack;
```

Comment:
Not sure what the "with track" suffix means here. Is this property really hoverThickness?

Reply:
Yeah. It's dependent on `showTrackWithHover`, I'll update and explain the behavior more in the docs.